docs/integrations/webhooks/sync-data.mdx

```diff
@@ -347,33 +271,35 @@ This guide can be adapted to listen for any Clerk event.
   - `SMSMessageJSON`
   - `UserJSON`
 
-  ### Test your webhook
+  ### Test the webhook
 
   1. Start your Next.js server.
-  1. On your endpoint's settings page in the Clerk Dashboard, select the **Testing** tab.
-  1. In the **Select event** dropdown, select `user.updated`.
-  1. Select the **Send Example** button.
-  1. Below that section, in the **Message Attempts** section, you should see a successful attempt with a status of `200`.
+  1. In your endpoint's settings page in the Clerk Dashboard, select the **Testing** tab.
+  1. In the **Select event** dropdown, select `user.created`.
+  1. Select **Send Example**.
+  1. In the **Message Attempts** section, confirm that the event is labeled with `Succeeded`.
 
-  #### Message failed
+  #### Handling failed messages
 
-  If the message failed:
-
-  1. Select the message.
+  1. In the **Message Attempts** section, select the event labeled with `Failed`.
   1. Scroll down to the **Webhook Attempts** section.
-  1. Select the arrow icon the left side.
-  1. Investigate the error. Your solution is going to depend on the error message. See the [Debug your webhooks](/docs/integrations/webhooks/debug-your-webhooks) guide for more information.
+  1. Toggle the arrow next to the **Status** column.
+  1. Review the error. Solutions vary by error type. For more information, refer to the [Debug your webhooks](/docs/integrations/webhooks/debug-your-webhooks) guide.
 
-  ### Trigger your webhook
+  ### Trigger the webhook
 
-  To trigger the `user.updated` event, you can do one of the following:
+  To trigger the `user.created` event, you can do either one of the following:
 
-  1. You can edit your user in the Dashboard.
-  1. You can use the `<UserProfile />` component in your application to edit your profile.
+  1. Edit your user in the Clerk Dashboard.
+  1. Select the `<UserProfile />` component in your app to edit your profile.
 
-  Once you have updated a user, you should be able to see the webhook's payload logged to your terminal. You can also check the Clerk Dashboard to see the webhook attempt, the same way you did when [testing the webhook](#test-your-webhook).
+  Once you have updated a user, you should be able to see the webhook's payload logged to your terminal. You can also check the Clerk Dashboard to see the webhook attempt, the same way you did when [testing the webhook](#test-the-webhook).
```

Comment:
Maybe not worth changing, but technically this guide doesn't actually update the user. Maybe just this?
```suggestion
  You should be able to see the webhook's payload logged to your terminal. You can also check the Clerk Dashboard to see the webhook attempt, the same way you did when [testing the webhook](#test-the-webhook).
```